docs/index.md

```diff
@@ -8,7 +8,7 @@ title: imPoster
 </p>
 
 <h1 class="post-title">{{ page.title | escape }}</h1>
-<h3 class="post-subtitle">v1.3</h3>
+<h3 class="post-subtitle">v1.4</h3>
```

Comment:
I think we can put v1.4b since this is the final milestone :D

Reply:
ok sure